src/types.ts

```diff
@@ -24,10 +24,12 @@ export type File = {
   path: string
 }
 
+type EngineType = 'library' | 'binary' | 'dataproxy'
 export type BaseTemplateParameters = {
   datasourceProvider?: Data.PrismaDatasourceProviderName
   repositoryOwner?: string | null
   repositoryHandle?: string | null
+  engineType?: EngineType | null
```

Comment:
Add some jsDoc, e.g.:
```suggestion
/**
 * The PSL prisma client generator block engineType setting.
 * @default '??'
 */
  engineType?: EngineType | null
```